doc/develop/test/bsim.rst

```diff
@@ -132,6 +132,14 @@ and then directly run one of the tests:
 Conventions
 ===========
 
+Characters
+----------
+
+Please don't use non-ASCII characters, like unicode or emoji in the scripts or test code.
```

Comment:
I agree with this, but it seems a bit out of place to have guidelines like this here.
(This should apply to the whole tree, not just to tests)

Reply:
will remove. I saw fabio is trying to add it to the compliance step anyways